### PR TITLE
Add DMP chapter

### LINKED ISSUE #2: Add  a chapter on DMP
Missing chapter, please add it!

### PR DESCRIPTION
**Pull request by chartgerink**
This Pull Request (PR) adds the data management plan chapter as provided in the Google Docs. 

There are several sections that need to be added and that can be a part of the review process.

Questions I have for reviewers:
- Do we want the sections to be numbered or unnumbered? 
- I did not yet include captions or alt text for the screenshots - do we want either/both?

I am inviting you all for reviews, but please do not feel obliged to all review 😊 Please also feel free to include any meta comments to the review process (e.g. preview website, automated link checks, etc.)

If merged, fixes #2.

### CODE REVIEW

File: _quarto.yml
Comment (Jolien-S) [inline _quarto.yml:1]: This looks good.

File: chapters/data-management-plan.qmd [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Data Management Plan
+
+## What is a DMP
+
+Data Management Plan (DMP) is a document outlining how research data will be handled throughout research lifecycle. A DMP is a structured way to address data collection, organization, storage, sharing, and preservation. It outlines the measures taken to ensure data security. DMP also address how data will be preserved and made available for future use.
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:5]: I'm missing some articles here. Could we add:

Line 5: 
- **A** Data Management Plan
- **the** research lifecycle
- DMP > DMP**s** (plural)

File: chapters/data-management-plan.qmd [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+| Processed data | Transcription of interviews | Word files      | Docx   |
+|                | Data spreadsheet            | SPSS files      | SAV    |
+| Analysed data  | Regression graphic          | Photoshop files | PSD    |
+|                | Data table                  | Word file       | Docx   |
+| Other          | Poster presentation         | Powerpoint      | PPS    |
+|                | Project Website             | HTML            |        |
+
+## DMP Elements
+
+[this is in the guidance on DMPonline, but we use this as a place where we can refer to ethics committees, privacy champions, etc. Make it clear that there is not necessarily a set order - sometimes there is, e.g. a DPIA may follow after a pre-DPIA indicates that]
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:95]: The part in square brackets may be deleted.

File: chapters/data-management-plan.qmd [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+The VU DMP template consists of seven sections with questions. In DMPonline, there is guidance available for all sections, as well as example answers. When you are writing your DMP, you can consult this information directly in DMPonline.
+
+### Data description
+
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:100]: We still need to decide if there is any relevant information that should go here.

File: chapters/data-management-plan.qmd [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+* ACTA: [ACTA Institutional Review Board](https://acta.nl/en/research/more-about/acta-institutional-review-board-irb) (IRB)
+* Beta: [Research ethics review committee Faculty of Science](https://vu.nl/en/about-vu/faculties/faculty-of-science/more-about/research-ethics-review-committee-beta) (BETHCIE)
+* FGB: [Scientific and Ethical Review Board](https://vu.nl/en/employee/behavioural-and-movement-sciences-getting-started/the-scientific-and-ethical-review-board-vcwe) (VCWE)
+* FGW: [Ethische Toetsingscommissie Onderzoek](https://vu.nl/en/about-vu/faculties/faculty-of-humanities/more-about/research-ethics-and-integrity) (EtCO)
+* FSW: [Research Ethics Review Committee](https://vu.nl/en/employee/social-sciences-getting-started/research-ethics-review-fss) (RERC)
+* RCH (Faculty of Law): [Ethics Committee](http://www.rechten.vu.nl/en/about-the-faculty/committees/ethics/index.aspx)
+* SBE: [Research Ethics and Integrity](https://vu.nl/en/about-vu/more-about/research-office)
+* VUmc: [METc](https://www.vumc.nl/research/overzicht/medisch-ethische-toetsingscommissie.htm) (Medical Ethical Review Committee)
+
+### Storage and backup during the research process
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:116]: We still need to decide if there is any relevant information that should go here.

File: chapters/data-management-plan.qmd [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+### Data archiving and publishing
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:117]: We still need to decide if there is any relevant information that should go here.

File: chapters/data-management-plan.qmd [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+### Documentation
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:118]: We still need to decide if there is any relevant information that should go here.

File: chapters/data-management-plan.qmd [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+### Data management responsibilities and resources
```
Comment (Jolien-S) [inline chapters/data-management-plan.qmd:119]: We still need to decide if there is any relevant information that should go here.

**Review by Jolien-S — APPROVED** [review]
Most of the content looks good. I have a few minor remarks. I've put these comments in the relevant lines of the text. The final section needs more work, we still need to look into this.

In reply to Chris' questions:
- Numbered or unnumbered sections: since faculties may use only parts of the resource, I can imagine numbered items don't make much sense.
- Captions/alt text: I'm not sure what the conventions are for captions in online pages. Alt text would be good to provide, I think. So when we're providing alt text anyway, we can use that for captions as well?

**Review by Karvovskaya — APPROVED** [review]
Most of the text is great; a few things need to be worked out

**Pull request merged**